neuraxle/base.py

```diff
@@ -637,7 +660,7 @@ def are_steps_before_index_the_same(self, other: 'TruncableSteps', index: int):
 
         return True
 
-    def _load_saved_pipeline_steps_before_index(self, saved_pipeline: 'TruncableSteps', index: int):
+    def _replace_pipeline_steps_before_index_by_other_pipeline_steps_before_index(self, saved_pipeline: 'TruncableSteps', index: int):
```

Comment:
Never use the word pipeline in a truncable step class. There is an LSP or ISP problem here.

Reply:
im not even using that old dead code

Reply:
So should it be removed?